### ISSUE TITLE
This plugin is not properly prepared for localization

### ISSUE THREAD

**Issue by patilswapnilv** [lubusIN/wordcamp-dashboard-widget#25]
This plugin is not properly prepared for localisation. 
Please take care of it so we can translate it.
This reference may help https://codex.wordpress.org/I18n_for_WordPress_Developers

**Comment by ajitbohra** [comment]
@patilswapnilv yes that's long due pending (#12) along with other tasks mentioned here https://github.com/lubusIN/wordcamp-dashboard-widget/projects/1 

Will soon get back to development & planning
